cloud_infra_center/ocp_upi/add-new-compute-node.yaml

```diff
@@ -0,0 +1,79 @@
+# =================================================================
+# Licensed Materials - Property of IBM
```

Comment:
I will check with our legal team but I do believe at that IBM surrenders ownership by use of Apache 2.0 License. I will get back to you on this.